tracer/src/emulator/mmu.rs

```diff
@@ -736,12 +769,30 @@ impl Mmu {
                 0x10001000..=0x10001FFF => panic!("store_raw:disk is unsupported."),
                 _ => {
                     self.assert_effective_store_address(effective_address);
-                    self.jolt_device.store(effective_address, value);
+                    self.jolt_device
+                        .as_mut()
+                        .expect("JoltDevice not set")
+                        .store(effective_address, value);
                 }
             },
         };
     }
 
+    pub fn setup_bytecode(&mut self, p_address: u64, value: u8) {
```

Comment:
added a new function so that we can load the bytecode into memory without triggering safety checks